### ISSUE TITLE
Enable disabling certain rules

### ISSUE THREAD

**Issue by MatthewDarling** [jonase/kibit#148]
One of Kibit's current rules is, "replace ```(filter seq coll)``` with ```(remove empty? coll)```". Some of the people on my team prefer ```(filter seq coll)```, so I'd like to disable that one rule, while using the rest of Kibit as usual.

From looking at [the code](https://github.com/jonase/kibit/blob/8424bb2/src/kibit/rules/misc.clj), it seems like there isn't a simple way to specify one individual rule for removal. I guess the user could provide literals for what rules to remove... Alternatively, they could be named and then have the removals operate on those names.

**Comment by danielcompton** [comment]
#139 kind of paves the way for this. I've been thinking about how best to allow people to do just what you're asking, #2 is another long standing issue that would be addressed with this.

**Comment by MatthewDarling** [comment]
Actually, I went and looked at [rules.clj](https://github.com/jonase/kibit/blob/8424bb2b20fd40f61dae27dd2c49c9f487ce3259/src/kibit/rules.clj) and it looks like we could get what we want by just copying the existing ```defrules``` and removing some. And then just use the new ```defrules``` as a custom rule set, and rely on ```merge``` to overwrite the old ones.

So it's possible, which is good for me to know! It's just not granular enough right now to swap out a small number of rules.

**Comment by danielcompton** [comment]
Yeah, there really needs to be identifiers for every rule (or collection of rules) which would let people specify rules to include/ignore. A PR on this would be amazing if you're inclined :)

**Comment by urzds** [comment]
I am also interested in seeing this, for the same reasons that @lvh stated in RackSec/desdemona#16.

**Comment by didibus** [comment]
Any hope for this anytime soon?

**Comment by acron0** [comment]
@danielcompton Any groundwork on this or would you take a PR?

**Comment by marksto** [comment]
Hi @danielcompton!

There are already 4 incoming PRs for this repo providing this feature (and the default ruleset exclusions):
- #139 (+ #141 w/ docs update) — custom rules-only
- #159 — exclusions-only
- #236 — both

And there is, as well, an unfinished attempt to have a fine-grained ruleset (as a map with IDs and descriptions) in the #176.

While the last one is unfinished (and for a good reason, since it's a huge effort requiring a solid Clojure background), other three are quite straightforward and (almost) equally ready to be approved and merged.

It would be great if any collaborator of this project will finally approve at least one of these PRs.
Then the community will stop providing the same stuff over and over again and can focus on improving what's there.

Cheers,
Mark
